data/json/items/gun/shot.json

```diff
@@ -211,7 +236,11 @@
     "durability": 7,
     "barrel_length": 3,
     "magazine_well": 1,
-    "magazines": [ [ "shot", [ "saiga10mag", "saiga30mag" ] ] ],
+    "magazines": [
+      [ "shot", [ "saiga10mag", "saiga30mag" ] ],
+      [ "308", [ "falmag", "falbigmag" ] ],
```

Comment:
Currently, the 223 and 308 conversion kits support converting a shotgun into a rifle. There appear to be some websites advertising Saiga-12 conversion kits; however, I haven't checked them out.